src/components/ContentResult.jsx

```diff
@@ -55,6 +57,11 @@ export const ContentResult = ({ contacts, allGroups }) => {
     translatedDefaultSelectedGroup(t)
   )
 
+  const handleSearchThreshold = ev => {
+    const thresholdValue = parseFloat(ev.target.value)
+    delayedSetThreshold(thresholdValue)
```

Comment:
why should we store it in the state? Since we have to do the search again to have new result? Storing it in flag('contacts.Threshold') will be enough, no ?

Reply:
On ne stock pas dans le state ici... ?